debian/control

```diff
@@ -0,0 +1,132 @@
+Source: bm
+Section: net
+Priority: optional
+Maintainer: Thomas Dreibholz <dreibh@simula.no>
+Build-Depends: automake,
+               cmake,
+               curl,
+               debhelper (>= 12),
+               dh-python,
+               git,
+               libboost-dev,
+               libboost-filesystem-dev,
+               libboost-program-options-dev,
+               libboost-system-dev,
+               libboost-thread-dev,
+               libgmp-dev,
+               libgrpc++-dev,
+               libgrpc-dev,
+               libjudy-dev,
+               libnanomsg-dev,
+               libpcap-dev,
+               libprotobuf-dev,
+               libprotoc-dev,
+               libssl-dev,
+               libthrift-dev (>= 0.13.0),
+               libtool,
+               pkg-config,
+               protobuf-compiler,
+               protobuf-compiler-grpc,
+               python3-all,
+               python3-six,
+               python3-thrift,
+               thrift-compiler
+Standards-Version: 4.5.1.0
+Rules-Requires-Root: no
+Homepage: https://github.com/simula/behavioral-model
+Vcs-Git: https://github.com/simula/behavioral-model
+Vcs-Browser: https://github.com/simula/behavioral-model/tree/p4-packaging
+
+Package: p4lang-bmv2-bin
+Architecture: amd64 arm64
```

Comment:
Has this been tested to work on arm64?

Reply:
The ARM64 builds pass the unit tests, which run as part of the package building procedure. This works for Ubuntu 20.04, 20.10 and 21.04. However, I did not make further ARM64 tests besides having the packages build for ARM64 in my Ubuntu Launchpad PPA.